hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -407,3 +407,33 @@ prepare_for_runner_image() {
   export OZONE_IMAGE="apache/ozone-runner:${v}"
 }
 
+## @description Executing the Ozone Debug CLI related robot tests
+execute_debug_tests() {
+
+  execute_robot_test datanode debug/ozone-debug-tests.robot
+
+  docker exec ozone_datanode_2 find /data/hdds/hdds/ -name *.block | sed -n '2p' | xargs docker exec ozone_datanode_2 sed -i -e '1s/^/a/'
```

Comment:
I think CI failed this time because maybe there are other keys created from other tests, so this might not be the right block file you are trying to corrupt.

Reply:
Alternatively you could try this tool to get the exact file .
`ozone debug chunkinfo ${VOLUME}/${BUCKET}/${TESTFILE} |  jq -r '.KeyLocations[0][0].Locations.files[0]`

Reply:
Yes, that was the problem, it wasn't the best way to do the corruption. Thanks, I fixed it with the use of chunkinfo.